src/rpc/mining.cpp

```diff
@@ -525,6 +525,7 @@ UniValue getblocktemplate(const JSONRPCRequest& request)
         // Need to update only after we know CreateNewBlock succeeded
         pindexPrev = pindexPrevNew;
     }
+    assert(pindexPrev && &pblocktemplate->block);
```

Comment:
Are you sure the analyzer doesn't realize a reference to an object is never null? Weird.

Reply:
@kallewoof Agreed – it should understand that. Checking `pindexPrev` should be sufficient. Updated. Please re-review :-)